src/connection/postgresql.py

```
@@ -0,0 +1,44 @@
+from sqlalchemy import create_engine as _create_engine
+from sqlalchemy.engine import (
+    URL as _URL,
```

Ruff (0.12.2)

`sqlalchemy.engine.URL` imported but unused

Remove unused import: `sqlalchemy.engine.URL`

(F401)